src/shortcutmanager.cpp

```diff
@@ -44,6 +44,7 @@ void ShortcutManager::initializeShortcutsList()
 {
     shortcutsList.append({tr("Open"), "open", keyBindingsToStringList(QKeySequence::Open), {}});
     shortcutsList.append({tr("Open URL"), "openurl", QStringList(QKeySequence(Qt::CTRL | Qt::SHIFT | Qt::Key_O).toString()), {}});
+    shortcutsList.append({tr("Reload File"), "reloadfile", keyBindingsToStringList(QKeySequence::Refresh), {}});
```

Comment:
indeed this is cmd+r on macOS, but this conflicts with rename. I don't know which should be prioritized--maybe refresh is more important. 

Make sure this doesn't conflict on other platforms too.

Reply:
Good catch, you can take a look at my latest commit and see if you like that idea or have a different suggestion perhaps.